changelog.d/8230.misc

```diff
@@ -0,0 +1 @@
+Track the latest event for every destination and room for catch-up after federation outage.
```

Comment:
once you land the other parts of this feature, it'll be good to update this so that the changelog entries get combined.